UnityProject/Assets/Scripts/Debug/TADB.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Runtime.CompilerServices;
+
+
+public class TADB_Debug
+	{
+	public enum Category{
+		Atmospherics,
+		Movement,
+		MapLoad, 
+		RoomControl,
+		ItemList,
+		ItemEntry,
+		DmiIconData,
+		UI,
+		MatrixManager,
+		Equipment,
+		NetworkManager,
+		uniCloth,
+		NetworkTabManager,
+		ItemFactory,
+		Light2D,
+		PlayerList,
+		PoolManager,
+		SpriteManager,
+		PlayerNetworkActions,
+		SteamIntegration,
+		TabUISystem,
+		Telecommunications,
+		PlayerJobManager,
+		PushPull,
+		NetSpriteImage,
+		ShutterController,
+		ShipNavigation,
+		Lighting,
+		Electrical, 
+	};
+	//-1 will not show anything
+	// 0 Only will show errors 
+	// 1 will show errors and warnings
+	// 2 will show errors, warnings and logs
+	private static Dictionary<string, int> NameDictionary = new Dictionary<string, int>{
```

Comment:
Suggest introducing default log level to config as well.
People absolutely shouldn't be forced to add categories here, it could rather list exceptions from default log level

Reply:
If it's not in the list of categories it just logs it anyway
"if (NameDictionary.ContainsKey(Category)) " 
"else
Debug_Level_process((Category+" "+LevelMessage[PriorityLevel]+Message+"\n"+ memberName), PriorityLevel);"

Reply:
In case it's unlisted it should check against the default logging level setting, I think that's reasonable